builds/fluent_bit.sh

```diff
@@ -14,26 +14,37 @@
 # limitations under the License.
 
 set -x -e
-DESTDIR=$1
+
+DESTDIR="$1"
 mkdir -p $DESTDIR
 fluent_bit_dir=/opt/google-cloud-ops-agent/subagents/fluent-bit
-  
+
+CMAKE_ARGS=$(cat <<-END
+-DCMAKE_BUILD_TYPE=RelWithDebInfo -DFLB_HTTP_SERVER=ON -DFLB_DEBUG=OFF \
+-DWITHOUT_HEADERS=ON -DFLB_SHARED_LIB=OFF -DFLB_STREAM_PROCESSOR=OFF \
+-DFLB_MSGPACK_TO_JSON_INIT_BUFFER_SIZE=1.5 -DFLB_MSGPACK_TO_JSON_REALLOC_BUFFER_SIZE=.10 \
+-DFLB_CONFIG_YAML=OFF
+END
+)
+
 cd submodules/fluent-bit
 mkdir -p build
 cd build
-# CMAKE_INSTALL_PREFIX here will cause the binary to be put at
-# /usr/lib/google-cloud-ops-agent/bin/fluent-bit
-# Additionally, -DFLB_SHARED_LIB=OFF skips building libfluent-bit.so
-cmake .. -DCMAKE_INSTALL_PREFIX=$fluent_bit_dir \
-  -DFLB_HTTP_SERVER=ON -DFLB_DEBUG=OFF -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-  -DWITHOUT_HEADERS=ON -DFLB_SHARED_LIB=OFF -DFLB_STREAM_PROCESSOR=OFF \
-  -DFLB_MSGPACK_TO_JSON_INIT_BUFFER_SIZE=1.5 -DFLB_MSGPACK_TO_JSON_REALLOC_BUFFER_SIZE=.10 \
-  -DFLB_CONFIG_YAML=OFF
-make -j8
-make DESTDIR="$DESTDIR" install
+if [[ "$DESTDIR" = *"dist" ]]; then
+  cmake .. $CMAKE_ARGS
+  make -j8
+  mv ./bin/fluent-bit $DESTDIR
+else
+  # CMAKE_INSTALL_PREFIX here will cause the binary to be put at
+  # /usr/lib/google-cloud-ops-agent/bin/fluent-bit
+  # Additionally, -DFLB_SHARED_LIB=OFF skips building libfluent-bit.so
```

Comment:
This comment should be with the `CMAKE_ARGS` assignment, right?

Reply:
Moved in [0572082](https://github.com/GoogleCloudPlatform/ops-agent/pull/1722/commits/0572082dbed8f1adcb3528416c92e54f4cd6e180)